modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -97,6 +94,16 @@ def enrollment
         end
       end
 
+      def send_confirmation_email
+        return unless Flipper.enabled?(:form1990meb_confirmation_email)
+
+        status = params[:claim_status]
+        email = params[:email] || @current_user.email
```

Comment:
Double checking for certainty: the correct method/attribute to retrieve the current user's email address is in fact `#email`? 

Asking because this has been an issue in the simple_forms_api and the correct method/attribute there is `va_profile_email`.

Reply:
Based on the code in `vets-website` I believe `email` is correct in this situation, although @bradbergeron-us or team may have other insight into it.

Reply:
based on higher level review, I'm alright with this as is.